recipe/meta.yaml

```diff
@@ -28,7 +28,14 @@ about:
   license: CC0-1.0
   license_family: CC
   license_file: LICENSE.md
-  summary: Open-source groundwater flow model
+  summary: USGS Modular Hydrologic Model
+  description: |
+    MODFLOW 6 is an object-oriented program and framework developed to provide
+    a platform for supporting multiple models and multiple types of models
+    within the same simulation. Within this framework, a regional-scale
+    groundwater model may be coupled with multiple local-scale groundwater
+    models. Or, a surface-water flow model could be coupled to multiple
```

Comment:
My only comment is the we actually do not support a surface-water flow model coupled to multiple groundwater models. Maybe it could be modified to "Or, a surface-water flow model (currently not available) could be coupled to multiple groundwater flow models."

Reply:
I'd prefer to keep the description to current capabilities, so I'll rephrase to mention groundwater transport instead.